src/main/java/seedu/address/logic/commands/InterviewCommand.java

```diff
@@ -0,0 +1,124 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Objects;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.person.InterviewDate;
+import seedu.address.model.person.Person;
+import seedu.address.model.person.exceptions.DuplicatePersonException;
+import seedu.address.model.person.exceptions.PersonNotFoundException;
+
+
+/**
+ * Schedule interview of an existing person in the address book.
+ */
+public class InterviewCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "interview";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Schedule interview for the person "
+            + "by the index number used in the last person listing. "
+            + "Existing scheduled date will be overwritten by the input value.\n"
+            + "Parameters: INDEX (must be a positive integer) "
+            + "DATETIME (parse by natural language)\n"
+            + "Example: " + COMMAND_WORD + " 1 next Friday at 3pm";
+
+    public static final String MESSAGE_INTERVIEW_PERSON_SUCCESS =
+            "Interview of person named %1$s has been scheduled on %2$s";
+    public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book.";
```

Comment:
Since changing the interview date will not result in duplicate person, this line can be removed?

Reply:
Thanks for the comment, I believe it depends on my implementation of `equals`, where in my case I treat both people with same interview date to be the same, but as @mhq199657 pointed out that it shouldn't be counted